rotkehlchen/chain/ethereum/modules/pickle/__init__.py

```diff
@@ -0,0 +1 @@
+from .pickle import PickleFinance  # noqa: F401
```

Comment:
nitpick: If it's just one file you don't really need to make an entire directory for the module